bundles/binding/org.openhab.binding.knx/src/main/java/org/openhab/binding/knx/internal/bus/KNXBusReaderScheduler.java

```diff
@@ -182,7 +185,7 @@ public synchronized boolean scheduleRead(Datapoint datapoint, int autoRefreshTim
 		if (oldListNumber > 0) { 
 			if (oldListNumber==autoRefreshTimeInSecs) {
 				sLogger.debug("Datapoint '{}' was already in  auto refresh list {}", datapoint.getName(), autoRefreshTimeInSecs);
-				return false;
+				return true;
```

Comment:
why did you change the logic here? There doesn't seem to be complains about an erroneous behaviour?